scripts/pureshear_ALE.jl

```diff
@@ -30,38 +30,52 @@ end
     return nothing
 end
 
+@parallel_indices (I...) function SetVelocity(V, verts, ε̇bg)
+
+    if I[1]<=size(V.x,1) &&  I[2]<=size(V.x,2)
+        V.x[I...] =  verts.x[I[1]]*ε̇bg
+    end
+
+    if I[1]<=size(V.y,1) &&  I[2]<=size(V.y,2)
+        V.y[I...] = -verts.y[I[2]]*ε̇bg
+    end
+
+return nothing
+end
+
+
+
 function main()
 
     @printf("Running on %d thread(s)\n", nthreads())
-    L  = (x=1., y=1.)
-    Nc = (x=41, y=41 )
-    Nv = (x=Nc.x+1,   y=Nc.y+1   )
-    Δ  = (x=L.x/Nc.x, y=L.y/Nc.y )
+    
+    # Parameters
+    L    = (x=1., y=1.)
+    Nc   = (x=41, y=41 )
+    Nv   = (x=Nc.x+1,   y=Nc.y+1   )
+    Δ    = (x=L.x/Nc.x, y=L.y/Nc.y )
     Nt   = 100
-    Nout = 1
+    Nout = 10
     C    = 0.25
 
+    # Model extent
     verts     = (x=LinRange(-L.x/2, L.x/2, Nv.x), y=LinRange(-L.y/2, L.y/2, Nv.y))
     cents     = (x=LinRange(-Δ.x/2+L.x/2, L.x/2-Δ.x/2, Nc.x), y=LinRange(-Δ.y/2+L.y/2, L.y+Δ.y/2-L.y/2, Nc.y))
     cents_ext = (x=LinRange(-Δ.x/2-L.x/2, L.x/2+Δ.x/2, Nc.x+2), y=LinRange(-Δ.y/2-L.y/2, L.y+Δ.y/2+L.y/2, Nc.y+2))
+    xlims     = [verts.x[1] , verts.x[end]]
+    ylims     = [verts.y[1] , verts.y[end]]
```

Comment:
```suggestion
    xlims     = verts.x[1] , verts.x[end]
    ylims     = verts.y[1] , verts.y[end]
```

Reply:
I don't think this is suitable:
```julia
julia> a = 1 , 1
(1, 1)

julia> typeof(a)
Tuple{Int64, Int64}
```
here we need a sector because these values should be mutable when the grid is deformed

Reply:
ah right I missed that point